### ISSUE TITLE
Doc date

### ISSUE THREAD

**Issue by jose-l-carvalho** [MicrosoftDocs/SupportArticles-docs#207]
[Enter feedback here]

Hi.
This issue appeared on June 2021, after update 6B was released.
But this doc date is March 31st, is it correct?

Thanks
---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: 84f27b89-2298-4df0-27c5-19360e0a557c
* Version Independent ID: 83e5144f-60c4-cd82-417e-62fdec52db89
* Content: [Azure VM is unresponsive with C01A001D error when applying Windows Update - Virtual Machines](https://docs.microsoft.com/en-us/troubleshoot/azure/virtual-machines/unresponsive-vm-apply-windows-update?source=docs)
* Content Source: [support/azure/virtual-machines/unresponsive-vm-apply-windows-update.md](https://github.com/MicrosoftDocs/SupportArticles-docs/blob/main/support/azure/virtual-machines/unresponsive-vm-apply-windows-update.md)
* Service: **virtual-machines**
* GitHub Login: @jc-mackin
* Microsoft Alias: **v-jcmackin**

**Comment by jc-mackin** [comment]
Yes, it’s correct. We rewrote an existing article so that it now emphasizes the new issue.

From: José Carvalho ***@***.***>
Sent: Thursday, July 15, 2021 10:57 AM
To: MicrosoftDocs/SupportArticles-docs ***@***.***>
Cc: JC Mackin (Infogain Corporation) ***@***.***>; Mention ***@***.***>
Subject: [MicrosoftDocs/SupportArticles-docs] Doc date (#207)


[Enter feedback here]

Hi.
This issue appeared on June 2021, after update 6B was released.
But this doc date is March 31st, is it correct?

Thanks
Document Details

⚠ Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.

  *   ID: 84f27b89-2298-4df0-27c5-19360e0a557c
  *   Version Independent ID: 83e5144f-60c4-cd82-417e-62fdec52db89
  *   Content: Azure VM is unresponsive with C01A001D error when applying Windows Update - Virtual Machines<https://nam06.safelinks.protection.outlook.com/?url=https%3A%2F%2Fdocs.microsoft.com%2Fen-us%2Ftroubleshoot%2Fazure%2Fvirtual-machines%2Funresponsive-vm-apply-windows-update%3Fsource%3Ddocs&data=04%7C01%7Cv-jcmackin%40microsoft.com%7C140dd7f0b04b4d75232808d947a0c53a%7C72f988bf86f141af91ab2d7cd011db47%7C1%7C0%7C637619578101849441%7CUnknown%7CTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7C1000&sdata=R53Axwyme32wgsIn4ONoXx%2B1lNwLAyqlsk7fQ9xaLwI%3D&reserved=0>
  *   Content Source: support/azure/virtual-machines/unresponsive-vm-apply-windows-update.md<https://nam06.safelinks.protection.outlook.com/?url=https%3A%2F%2Fgithub.com%2FMicrosoftDocs%2FSupportArticles-docs%2Fblob%2Fmain%2Fsupport%2Fazure%2Fvirtual-machines%2Funresponsive-vm-apply-windows-update.md&data=04%7C01%7Cv-jcmackin%40microsoft.com%7C140dd7f0b04b4d75232808d947a0c53a%7C72f988bf86f141af91ab2d7cd011db47%7C1%7C0%7C637619578101859565%7CUnknown%7CTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7C1000&sdata=g%2FAzNyNb5lUstqmDwc3XZ7F9wOfMJH6EukGTz6XRYEk%3D&reserved=0>
  *   Service: virtual-machines
  *   GitHub Login: @jc-mackin<https://nam06.safelinks.protection.outlook.com/?url=https%3A%2F%2Fgithub.com%2Fjc-mackin&data=04%7C01%7Cv-jcmackin%40microsoft.com%7C140dd7f0b04b4d75232808d947a0c53a%7C72f988bf86f141af91ab2d7cd011db47%7C1%7C0%7C637619578101869427%7CUnknown%7CTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7C1000&sdata=1utOC0LkOYf4dv2kTon5JizoWb8haNY2gW2UNow3XyE%3D&reserved=0>
  *   Microsoft Alias: v-jcmackin

**Comment by jocarv-fte** [comment]
Hey..

Thanks for the update. Meanwhile I was mad aware of that.

One strange thing that happens is that comments to either of the docs, fall on same github page 😊


(Next OOF: August 9th)
﻿Please use “reply all” when replying, to ensure proper case routing.


Best Regards,

José Carvalho
SUPPORT ESCAL ENG
Azure Rapid Response Portugal
Customer Services and Support
Office: (+351) 210602486
***@***.******@***.***>
Working hours: Mon-Fri 8:30-17:30 (UTC/UTC+1 in DST)
[Microsoft]

If you need immediate attention for any of the ongoing case while I am out of shift, please send a separate email with case ID to ***@***.******@***.***>. A duty manager will take action and loop in an available ARR engineer to help you. 
If you have any feedback about my work, please let either me or my manager Sofia Alves know at ***@***.******@***.***>

**Comment by genlin** [comment]
@jocarv-fte Thanks for the feedback. You are right. The publish date of this article is not correct. We will update it.
